src/pytest_cov/plugin.py

```diff
@@ -263,20 +263,24 @@ def pytest_terminal_summary(self, terminalreporter):
 
         terminalreporter.write('\n' + self.cov_report.getvalue() + '\n')
 
-        if self.options.cov_fail_under is not None and self.options.cov_fail_under > 0:
-            if self.cov_total < self.options.cov_fail_under:
+        fail_under = self.options.cov_fail_under
+        if fail_under is not None and fail_under > 0:
```

Comment:
btw, this could be just `if fail_under`, right?

Reply:
Hmmm ... I suspect it was written like that to allow users to disable fail under by repeating the option with 0 as argument.